cmd/cli/plugin/pinniped-auth/login_test.go

```diff
@@ -67,6 +67,7 @@ func TestLoginOIDCCommand(t *testing.T) {
 				"--skip-browser=false",
 				fmt.Sprintf("--session-cache=%s", sessionsCacheFilePath),
 				"--debug-session-cache=false",
+				fmt.Sprintf("--credential-cache=#{credentialCachePath}"),
```

Comment:
So I think this should look more like `session-cache` above it.  Also, we'll need to add this into the `wantArgs` for all the test below it.  It'd be great if we could add a custom path into `args` for the `test options are populated correctly with given user values` test as well.  Also, order matters for the check between actual/expected for the `wantArgs`, so if you get a failure there after adding the `--credential-cache` arg, it may just be due to order.

Reply:
Oh yeah, let me fill this out a little better.